### PR TITLE
Fix typos in MIGRATION.md

### PR DESCRIPTION
**Pull request by huytd**
I noticed there's some small typos in MIGRATION.md.

Those are `crate` and not `create`.

### CODE REVIEW

File: MIGRATION.md [hunk 1]
```diff
@@ -360,7 +360,7 @@
 
 * `actix_web::server` module has been removed. To start http server use `actix_web::HttpServer` type
 
-* StaticFiles and NamedFile has been move to separate create.
+* StaticFiles and NamedFile has been move to separate crate.
```
Comment (JohnTitor) [inline MIGRATION.md]: Let's fix more things.
```suggestion
* StaticFiles and NamedFile have been moved to separate crates.
```

Comment (huytd) [reply MIGRATION.md]: Sure, let me fix it. Thank you so much!

**Review by robjtede — APPROVED** [review]

**Comment by JohnTitor** [comment]
Thanks!

**Comment by huytd** [comment]
Thank you so much!

**Pull request merged**